### PR TITLE
Feature/send message

### PR DESCRIPTION
**Pull request by MatiasCab**
#### Board:
* [Ticket #Send a message](https://trello.com/c/mfX7MPYP)
---
#### Description:
 As a user I should be able to send a message in a conversation with any of my compatible targets so I can talk about our common interests 
---
#### Notes:
*
---
#### Tasks:
- [x] Add each element in this format
---
#### Risk:
*
---

### CODE REVIEW

**Review by smerlo — APPROVED** [review]
Nice work!!!🙌

File: app/services/create_message_service.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class CreateMessageService
+  def initialize(params, current_user)
+    @current_user = current_user
+    @params = params
+  end
+
+  def create!
+    message_info = {
+      content: resource_params[:content],
+      user_id: @current_user.id
```
Comment (enzofab91) [inline app/services/create_message_service.rb]: As a comment, you can do `attr_reader :current_user`, so here you don't need to use class variables.

**Review by enzofab91 — APPROVED** [review]
Nice work! 💯

File: app/services/create_message_service.rb (hunk 1 above)
Comment (MatiasCab) [reply app/services/create_message_service.rb]: Ok!

**Pull request merged**